### ISSUE TITLE
New Help-Viewer not working for some non {base} packages

### ISSUE THREAD

**Issue by assignUser** [Ikuyadeu/vscode-R#485]
I can no longer open help files for non {base} packages via e.g. `?insertUI` nor `?shiny::insertUI`. 
`r.showHelp`also does not work. Both open the viewer like this:
![image](https://user-images.githubusercontent.com/16141871/101021824-b721ff80-3570-11eb-8b9f-1cfd8ea22d8f.png)

The Help-Viewer works (and looks great!) for built in packages. I have tested it with several different packages that all fail: dplyr, shiny, simstudy, shinyjs. Loaded and unloaded.

For some reason it works with purrr, glue, base64enc. Also work Loaded without the package name and unloaded with package name. Not sure why these packages are different.

Let me know if you need further info!

 - OS: win 10
 - VSCode Version: 1.51.1
 - R Version: 4.0.2
 - vscode-R version: 1.6.1 with radian

**Comment by renkun-ken** [comment]
Looks like your `.libPaths` is not correctly loaded by the background R session started by https://github.com/Ikuyadeu/vscode-R/blob/master/src/rHelpProviderBuiltin.ts#L47.

**Comment by assignUser** [comment]
ah that is probably it, I just noticed that I have multiple libs I did not know about. I will set them in my Rprofile and check if that fixes things :)

**Comment by assignUser** [comment]
I looked into it some more my `.libPaths` only contains the library in my home directory and the `.Library` the R home directory (which is default behavior as I understand?). All non base packages are in the home library. As it works for some like  `purrr` the path should be detected correctly.

Any ideas on how to debug this would be appreciated 😺

**Comment by ManuelHentschel** [comment]
> Any ideas on how to debug this would be appreciated 😺

I think the best way to figure out what is going on would be to run the extension from code (clone the repo, run npm install, press F5) and set a breakpoint where the [help server is launched](https://github.com/Ikuyadeu/vscode-R/blob/master/src/rHelpProviderBuiltin.ts#L47). Then check the values of `cmd` and `cpOptions.cwd` and manually launch an R session in the directory specified by `cpOptions.cwd` (which should be the workspace folder) and with the executable (and commandline args except for `-e ...`) specified in `cmd`. There you can check `.libPaths`, `installed.packages()`, manually call `?`, etc. to get an idea of what's going on.

**Comment by assignUser** [comment]
ok will do when I have some time :)

**Comment by assignUser** [comment]
Happy Holidays!  

I have found the cause for this issue. I had not set an vanilla rPath in the settings, only one for radian,  and for some reason the R version and path in my registry was out of date.   

The extension generated the correct request path etc. but the help server was started in an old version that only had some of the packages I have installed in the 4.x library which explains the erratic behavior of this error.
